bundles/org.openhab.binding.automower/README.md

```diff
@@ -0,0 +1,123 @@
+
+## Supported Things
+
+`bridge:` The bridge needs to be configured with credentials and an application key that allows communicating with the Automower Connect Api
+
+`automower:` A single Husqvarna Automower robot
+
+Basically all Husqvarna Automower models with "Automower Connect" support should be supported. It was tested only with a Husqvarna Automower 450X
+
+
+## Discovery
+
+Once the bridge is created and configured, registered automowers will be discovered automatically
+
+
+## Thing Configuration
+
+`bridge:`
+
+- appKey (mandatory): The Application Key is required to communication with the Automower Connect Api. It can be obtained by registering an Application on the Husqvarna Website. This application also needs to be connected to the "Authentication API" and the "Automower Connect API"
+- userName (mandatory): The user name for which the application key has been issued
+- password (mandatory): The password for the given user
+- pollingInterval (optional): How often the bridge state should be queried in seconds. Default is 1h (3600s)
+
+Keep in mind that the status of the bridge should not be queried too often. According to the Husqvarna documentation not more than 10000 requests per month and application key are allowed. With the default value of 1h this would mean ~720 requests per month for the bridge state
+
+`automower:`
+
+- mowerId (mandatory): The Id of an automower as used by the Automower Connect Api to identify a mower. This is automatically filled when the thing is discovered
+- pollingInterval (optional): How often the current automower state should be polled in seconds. Default is 10min (600s)
+
+Keep in mind that the status of the automowers should not be queried too often. According to the Husqvarna documentation not more than 10000 requests per month and application key are allowed. With the default value of 10min this would mean ~4300 requests per month per automower
+
+## Channels
+
+
+| channel  | type   | description                  |
+|----------|--------|------------------------------|
+| name  | String | (readonly) The name of the Automower  |
+| mode  | String | (readonly) The current mode (MAIN_AREA, SECONDARY_AREA, HOME, DEMO, UNKNOWN)  |
+| activity  | String | (readonly) The current activity (UNKNOWN, NOT_APPLICABLE, MOWING, GOING_HOME, CHARGING, LEAVING, PARKED_IN_CS, STOPPED_IN_GARDEN)  |
+| state  | String | (readonly) The current state (UNKNOWN, NOT_APPLICABLE, PAUSED, IN_OPERATION, WAIT_UPDATING, WAIT_POWER_UP, RESTRICTED, OFF, STOPPED, ERROR, FATAL_ERROR, ERROR_AT_POWER_UP)  |
+| last-update  | DateTime | (readonly) The time when the automower updated its states  |
+| battery  | Number | (readonly) The battery state of charge in percent  |
+| error-code  | Number | (readonly) The current error code  |
+| error-timestamp  | DateTime | (readonly) The timestamp when the current error occurred  |
```

Comment:
These could be trigger Channels, since they don't have a state, but send events when an error raised.

Reply:
Ok, I did not see those channels as trigger channels. I changed the command channels, but the error code reflects the error code that is reported by the mower in case it is in an error state.

Reply:
After thinking about it again, I agree with you, that it is a state if the mower has an error.